libraries/c_sdk/standard/mqtt/include/mqtt_config.h

```diff
@@ -22,6 +22,13 @@
 
 /************ End of logging configuration ****************/
 
+/* Set network context used for sending the packets on the network. */
+typedef struct NetworkContext
+{
+    uint8_t ** networkConnection;
+    uint8_t ** networkInterface;
```

Comment:
Can't we directly use the networkInterface and networkConnection types here directly?